MobileCenterCrashes/MobileCenterCrashes/Internals/MSErrorAttachmentLog.m

```diff
@@ -77,7 +75,7 @@ - (NSMutableDictionary *)serializeToDictionary {
   if (self.contentType) {
     dict[kMSContentType] = self.contentType;
   }
-  if (self.filename) {
+  if (self.filename && self.filename.length > 0) {
```

Comment:
We discussed with the team about allowing empty strings in the payload and we do allow empty string there. Could you please remove the `&& self.filename.length > 0`?